build-system/tasks/e2e/puppeteer-controller.js

```diff
@@ -555,9 +555,10 @@ class PuppeteerController {
    */
   getTitle() {
     const title = this.getCurrentFrame_().then((frame) => frame.title());
```

Comment:
Can we remove the `.then()` here as well?

```suggestion
	const frame = await this.getCurrentFrame_();
    const title = frame.title();
```

Reply:
No, because the type of `title` is `Promise<string>`, which is what's required by `ControllerPromise`. The above would produce just the `string`. The promise setup here appears to be WAI